cni/network/network.go

```diff
@@ -781,8 +781,12 @@ func (plugin *NetPlugin) createEndpointInternal(opt *createEndpointInternalOpt)
 		NATInfo:            opt.natInfo,
 	}
 
-	isIPv6Enabled := opt.resultV6 != nil
-	epPolicies := getPoliciesFromRuntimeCfg(opt.nwCfg, isIPv6Enabled)
+	epPolicies, err := getPoliciesFromRuntimeCfg(opt.nwCfg)
+	if err != nil {
+		logger.Error("failed to get policies from runtime configurations", zap.Error(err))
+		return epInfo, err
```

Comment:
im surprised the linter doesn't complain about not wrapping the error

Reply:
This is using zap logger here

Reply:
it's not about logging, it's about the error returned, unless that linter is not enabled for checking that errors are wrapped? @rbtr

Reply:
can we wrap this error before returning?